### ISSUE TITLE
[Bug]: Error in checking FASTQ and Script

### ISSUE THREAD

**Issue by phongphak** [epi2me-labs/wf-bacterial-genomes#9]
### What happened?

Hi, I got an error while running wf-bacterial-genomes using demo data. However, I successfully executed the others such as wf-transcriptomes and wf-metagenomics without any issues. I also attempted to resolve the issue by deleting and re-installingl the workflow, but it still doesn't work. 

nextflow.log
https://drive.google.com/file/d/1U9TybH7EfQaYLwebU-Y7O3UKQVSF29X_/view?usp=share_link

Could you please help me with this?

### Operating System

Windows 11

### Workflow Execution

EPI2ME Labs desktop application

### Workflow Execution - EPI2ME Labs Versions

4.1.3

### Workflow Execution - CLI Execution Profile

Docker

### Workflow Version

0.2.13

### Relevant log output

```shell
N E X T F L O W  ~  version 22.04.5
Launching `/mnt/wsl/docker-desktop-bind-mounts/Ubuntu/3c273db223050d5cc07c2940af95c7595e6656d16e5332b9b39e30b9c8ec06cc/workflows/epi2me-labs/wf-bacterial-genomes/main.nf` [blissful_aryabhata] DSL2 - revision: 85754a734d
||||||||||   _____ ____ ___ ____  __  __ _____      _       _
||||||||||  | ____|  _ \_ _|___ \|  \/  | ____|    | | __ _| |__  ___
|||||       |  _| | |_) | |  __) | |\/| |  _| _____| |/ _` | '_ \/ __|
|||||       | |___|  __/| | / __/| |  | | |__|_____| | (_| | |_) \__ \
||||||||||  |_____|_|  |___|_____|_|  |_|_____|    |_|\__,_|_.__/|___/
||||||||||  wf-bacterial-genomes v0.2.13
--------------------------------------------------------------------------------
Core Nextflow options
  runName        : blissful_aryabhata
  containerEngine: docker
  launchDir      : /mnt/wsl/docker-desktop-bind-mounts/Ubuntu/3c273db223050d5cc07c2940af95c7595e6656d16e5332b9b39e30b9c8ec06cc/instances/wf-bacterial-genomes_082be453-ed96-4e18-8f7f-7c0fe5725e46
  workDir        : /mnt/wsl/docker-desktop-bind-mounts/Ubuntu/3c273db223050d5cc07c2940af95c7595e6656d16e5332b9b39e30b9c8ec06cc/instances/wf-bacterial-genomes_082be453-ed96-4e18-8f7f-7c0fe5725e46/work
  projectDir     : /mnt/wsl/docker-desktop-bind-mounts/Ubuntu/3c273db223050d5cc07c2940af95c7595e6656d16e5332b9b39e30b9c8ec06cc/workflows/epi2me-labs/wf-bacterial-genomes
  userName       : jump
  profile        : standard
  configFiles    : /mnt/wsl/docker-desktop-bind-mounts/Ubuntu/3c273db223050d5cc07c2940af95c7595e6656d16e5332b9b39e30b9c8ec06cc/workflows/epi2me-labs/wf-bacterial-genomes/nextflow.config, /mnt/wsl/docker-desktop-bind-mounts/Ubuntu/3c273db223050d5cc07c2940af95c7595e6656d16e5332b9b39e30b9c8ec06cc/instances/wf-bacterial-genomes_082be453-ed96-4e18-8f7f-7c0fe5725e46/demo.config
Input Options
  fastq          : /mnt/wsl/docker-desktop-bind-mounts/Ubuntu/3c273db223050d5cc07c2940af95c7595e6656d16e5332b9b39e30b9c8ec06cc/demo/epi2me-labs/wf-bacterial-genomes/wf-bacterial-genomes-demo/fastq
Output Options
  out_dir        : /mnt/wsl/docker-desktop-bind-mounts/Ubuntu/3c273db223050d5cc07c2940af95c7595e6656d16e5332b9b39e30b9c8ec06cc/instances/wf-bacterial-genomes_082be453-ed96-4e18-8f7f-7c0fe5725e46/output
!! Only displaying parameters that differ from the pipeline defaults !!
--------------------------------------------------------------------------------
If you use epi2me-labs/wf-bacterial-genomes for your analysis please cite:
* The nf-core framework
  https://doi.org/10.1038/s41587-020-0439-x
--------------------------------------------------------------------------------
This is epi2me-labs/wf-bacterial-genomes v0.2.13.
--------------------------------------------------------------------------------
Checking fastq input.
Input directory '/mnt/wsl/docker-desktop-bind-mounts/Ubuntu/3c273db223050d5cc07c2940af95c7595e6656d16e5332b9b39e30b9c8ec06cc/demo/epi2me-labs/wf-bacterial-genomes/wf-bacterial-genomes-demo/fastq' cannot contain FASTQ files and sub-directories with FASTQ files.
 -- Check script '/mnt/wsl/docker-desktop-bind-mounts/Ubuntu/3c273db223050d5cc07c2940af95c7595e6656d16e5332b9b39e30b9c8ec06cc/workflows/epi2me-labs/wf-bacterial-genomes/./lib/fastqingress.nf' at line: 293 or see '/mnt/wsl/docker-desktop-bind-mounts/Ubuntu/3c273db223050d5cc07c2940af95c7595e6656d16e5332b9b39e30b9c8ec06cc/instances/wf-bacterial-genomes_082be453-ed96-4e18-8f7f-7c0fe5725e46/nextflow.log' file for more details
```

**Comment by sarahjeeeze** [comment]
Hi, Sorry for that, this has been fixed in the latest release v0.2.14

**Comment by sarahjeeeze** [comment]
Closing through lack of response.
